src/opaque-types-impl-trait-inference.md

```diff
@@ -216,15 +216,15 @@ and then handle it correctly.
 The MIR borrow checker relates things via `nll_relate` and only cares about regions.
 Any type relation will trigger the binding of hidden types,
 so the borrow checker is doing the same thing as the type checker,
-but ignores obivously dead code (e.g. after a panic).
+but ignores obviously dead code (e.g. after a panic).
 The borrow checker is also the source of truth when it comes to hidden types,
 as it is the only one who can properly figure out what lifetimes on the hidden type correspond
 to which lifetimes on the opaque type declaration.
 
 ## Backwards compatibility hacks
 
 `impl Trait` in return position has various quirks that were not part
-of any RFCs and are likely accidental stabilizations.
+of any RFCs and are likely accidental stabilization.
```

Comment:
```suggestion
of any RFCs and are likely an accidental stabilization.
```
is this what was meant

Reply:
I'm not sure but does it become a "countable noun" in this context?

Reply:
hm, I think it would instead have to be `... and is likely an accidental stabilization.`

Reply:
Well, I'm talking about if it's a countable noun or an uncountable one and I think it depends on the context, but I'm not sure if this is the case.

Reply:
I don't know... it just didn't read well, or was not obvious, hence the suggested edit.